.github/workflows/nix.yaml

```
@@ -99,23 +68,93 @@ jobs:
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Push images and create manifest
         run: |
-          echo "$GITHUB_TOKEN" | docker login ghcr.io -u "${{ github.actor }}" --password-stdin
-          docker load < docker-image.tgz
-          docker push ${{ needs.build.outputs.docker-image-name }}
-          docker logout ghcr.io
+          # Load both images
+          docker load < amd64/docker-image-amd64.tgz
+          docker load < arm64/docker-image-arm64.tgz
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)